libr/core/rvc.c

```diff
@@ -713,6 +713,7 @@ R_API bool r_vc_commit(const char *rp, const char *message, const char *author,
 		eprintf ("Can't commit");
 		return false;
 	}
+	author = author? author : r_config_get (core->config, "prj.vc.author");
```

Comment:
Instead of calling config and core here just do this in the caller. So rvc stays independent